src/content/docs/infrastructure/host-integrations/host-integrations-list/active-directory-integration.mdx

```diff
@@ -26,50 +23,71 @@ Built with our infrastructure agent, the Active Directory integration gives you
+
+<Callout variant="tip">
+
+**Use guided install to quickly see your data in the UI**
+
+The guided install is a single CLI command you can run to monitor your Active Directory services. It's a good option for small organizations, or for anyone who wants to test out New Relic.
+
+<ButtonGroup>
+  <ButtonLink
+    role="button"
+    to="https://one.newrelic.com/marketplace?state=8f14e646-461e-b010-4675-3a0658bb3d20"
+    variant="primary"
+  >
+    Guided install
+  </ButtonLink>
+</ButtonGroup>
+
+For a more permanent and scalable solution, we recommend the standard manual install of the integrations: keep reading for how to do that.
+
+</Callout>
+
+
+<Steps>
+    <Step>
+
 ## Install the infrastructure agent [#infra]
 
-To see your Active Directory data in New Relic, install our infrastructure agent. The infrastructure agent collects data from Performance Counters, Windows Services, PowerShell scripts, and Windows Event Logs and sends that data to New Relic, where you can use the telemetry to measure the health of your Active Directory environment.. 
+To use the Active Directory integration, you need to first [install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/get-started/install-infrastructure-agent-new-relic/) on the same host. The infrastructure agent monitors the host itself, while the integration you'll install in the next step extends your monitoring with Active Directory-specific data.
+        
+    </Step>
+    <Step>
 
-There are two methods to install the infrastructure agent:
+## Clone the Active Directory integration repo [#clone]
 
-* Follow our [guided install](https://one.newrelic.com/marketplace?state=8f14e646-461e-b010-4675-3a0658bb3d20) to instrument your system with the infrastructure agent. 
-* [Install the infrastructure agent](/docs/infrastructure/install-infrastructure-agent/linux-installation/install-infrastructure-monitoring-agent-linux) via the command line using our docs.
+Run the following command in your CLI to clone the New Relic Active Directory integration repository:
 
-## Instrument Active Directory with New Relic [#instrument]
+```bash
+git clone https://github.com/newrelic/newrelic-active-directory-integration.git
+```        
 
-To instrument Active Directory with the infrastructure agent, follow the below procedures between this doc and the [New Relic Active Directory integration repository](https://github.com/newrelic/newrelic-active-directory-integration). 
+    </Step>
+    <Step>
 
-<CollapserGroup>
-    <Collapser
-    id="step1"
-    title="Step 1. Clone the repository"
-    >
-    Run the following command in your CLI to clone the New Relic Active Directory integration repository:
+## Copy configuration files [#copy-config]
+Copy the relevant configuration files from the repository to the agent, replacing `$ROOT_PATH` with the path where you cloned the repository:
```

Comment:
looks like "relevant configuration files" refers to the folders "integrations d." and "loogging.d". i might link out to these folders and just identify them specifically, rather than saying "relevant config files." Was a little confusing at first and wasn't sure if i was missing something comparing these instructions to the repo